apps/eo/e2e-eo/src/e2e/dashboard/dashboard.feature

```diff
@@ -3,6 +3,7 @@ Feature: Dashboard Page
   Background: The dashboard page is the first page a user gets to see after login, and it should have the most pressing
   items for a user to see, and then feel inclined to visit the rest of the pages to get more in-depth information.
 
+  @skip
```

Comment:
We're temporarily skipping the existing `e2e` tests because they log in like regular users. Since we haven't optimized the performance until [#2704](https://app.zenhub.com/workspaces/team-atlas-633199659e255a37cd1d144f/issues/gh/energinet-datahub/energy-origin-issues/2704), the tests are flaky and sometimes time out. They will be enabled back again or changed to more proper tests in the mentioned issue.